packages/carbon-web-components/src/components/content-switcher/content-switcher.ts

```diff
@@ -12,7 +12,7 @@ import { property, customElement } from 'lit/decorators.js';
 import { prefix } from '../../globals/settings';
 import { forEach, indexOf } from '../../globals/internal/collection-helpers';
 import { NAVIGATION_DIRECTION, CONTENT_SWITCHER_SIZE } from './defs';
-import BXSwitch from './content-switcher-item';
+import CDSSwitch from './content-switcher-item';
```

Comment:
```suggestion
import CDSContentSwitcherItem from './content-switcher-item';
```
Should this be changed?

Reply:
I wasn't sure about this because the name in React is still Switch and IconSwitch so I kept it

Reply:
ahh okay yeah this is a weird one... since we don't have the same switch or iconswitch in wc just contentSwitcherItem